lib/Transforms/UnifyKernels.cpp

```diff
@@ -0,0 +1,331 @@
+/*
+ * Copyright HeteroCL authors. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <iostream>
```

Comment:
I guess this is for testing purposes. You can remove this library

Reply:
> I guess this is for testing purposes. You can remove this library

done